src/index.js

```diff
@@ -10,6 +10,7 @@ import store from 'state/store';
 import App from './App';
 import locales from './locales';
 import './index.css';
+import '../node_modules/primeflex/primeflex.css';
```

Comment:
Try importing primeflex.css in the css file itself using

`@import ~primeflex/primeflex.css`

Reply:
Will remove it as I didn't end up using it yet